Tests/SwiftGRPCNIOTests/GRPCInteroperabilityTests.swift

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2019, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import Foundation
+import SwiftGRPCNIO
+import SwiftGRPCNIOInteroperabilityTests
+import NIO
+import XCTest
+
+/// These are the gRPC interoperability tests running on the NIO client and server.
+class GRPCInsecureInteroperabilityTests: XCTestCase {
+  var useTLS: Bool { return false }
+  var defaultTestTimeout: TimeInterval = 5.0
+
+  var serverEventLoopGroup: EventLoopGroup!
+  var server: GRPCServer!
+
+  var clientEventLoopGroup: EventLoopGroup!
+  var clientConnection: GRPCClientConnection!
+
+  override func setUp() {
+    super.setUp()
+
+    self.serverEventLoopGroup = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+    self.server = try! makeInteroperabilityTestServer(
+      host: "localhost",
+      port: 0,
+      eventLoopGroup: self.serverEventLoopGroup!,
+      useTLS: self.useTLS
+    ).wait()
+
+    guard let serverPort = self.server.channel.localAddress?.port else {
+      XCTFail("Unable to get server port")
+      return
+    }
+
+    self.clientEventLoopGroup = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+    self.clientConnection = try! makeInteroperabilityTestClientConnection(
+      host: "localhost",
+      port: serverPort,
+      eventLoopGroup: self.clientEventLoopGroup,
+      useTLS: self.useTLS
+    ).wait()
+  }
+
+  override func tearDown() {
+    XCTAssertNoThrow(try self.clientConnection.close().wait())
+    XCTAssertNoThrow(try self.clientEventLoopGroup.syncShutdownGracefully())
+    self.clientConnection = nil
```

Comment:
nit: just to be sure, would you mind setting the event loop groups to `nil` as well?